pkg/features/kube_features.go

```diff
@@ -662,6 +662,13 @@ const (
 	// Requires the CRI implementation supports supplying the required stats.
 	PodAndContainerStatsFromCRI featuregate.Feature = "PodAndContainerStatsFromCRI"
 
+	// owner: @alculquicondor
+	// deprecated: v1.24
+	//
+	// Enables preconditions checks for UID and resourceVersion in the pod/binding
```

Comment:
just UID, right? resourceVersion appears to have worked previously

Reply:
can you keep the feature-gate commit separate from the picked commits?

Reply:
if this is a regression (you linked to https://github.com/kubernetes/kubernetes/pull/110646 which is where this regressed), then I don't think we need the feature gate and can just backport the fix